bullet.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+### THIS CODE MADE FOR STAY IN TARKOV AND LICENSED UNDER MIT
+
+SIT_VERSION="${SIT:=latest}"
+HEADLESS="${HEADLESS:-}"
+FORCE="${FORCE:-}"
+echo "Stay In Tarkov Docker"
+echo "github.com/StayInTarkov"
+
+if [ ! -e "/opt/server/version" ] || [ ! -z "$FORCE" ]; then
+  if [ -d "/opt/srv" ]; then
+    start=$(date +%s)
+    echo "Started copying files to your volume/directory.. Please wait."
+    cp -r /opt/srv/* /opt/server/
+    rm -r /opt/srv
+    end=$(date +%s)
+    echo "Files copied to your machine in $(($end-$start)) seconds."
+    echo "Starting the server to generate all the required files.. Please wait."
+    cd /opt/server
+    chown $(id -u):$(id -g) ./* -Rf
+    EXT_IP=$(curl -4 ipv4.icanhazip.com)
+    sed -i 's/\"ip\": \"127.0.0.1\"/\"ip\": \"0.0.0.0\"/g' ./Aki_Data/Server/configs/http.json
+    sed -i "s/\"backendIp\": \"127.0.0.1\"/\"backendIp\": \"$EXT_IP\"/g" ./Aki_Data/Server/configs/http.json
+    NODE_CHANNEL_FD= timeout 40s ./Aki.Server.exe </dev/null >/dev/null 2>&1
```

Comment:
Just checking intent here, this is running `AKI` for up to 40s to make sure things boot up correctly after setup & generate some logs? Seems healthy. Did you mean to do something after this with `NODE_CHANNEL_FD`?